src/main/java/seedu/address/model/note/Note.java

```diff
@@ -30,10 +30,10 @@ public boolean equals(Object obj) {
         return false;
     }
 
-    //  @Override
-    //  public String toString() {
-    //      return noteContents;
-    //  }
+    @Override
+    public String toString() {
+        return noteContents;
+    }
 
     //  @Override
     //  public int hashCode() {
```

Comment:
There is a need to implement the hashCode function, since note is being used in a hashSet.
Could you also change the equals method to match the 3 step thing? (1. short circuit if same object. 2. instanceof handles nulls 3. state check)

Reply:
updated with an autogenerated hashCode()